winit/src/application.rs

```diff
@@ -101,12 +102,6 @@ impl Application {
         self
     }
 
-    pub fn locale(mut self, id: &str) -> Self {
```

Comment:
This is something we can revisit later but I think it would be good to be able to set the locale for testing purposes.